tests/test_meshtastic_connection.py

```
@@ -317,6 +317,279 @@ def test_tcp_connection_returns_none_client_raises_connection_error(self):
+
+        assert result is None
+        mock_disconnect.assert_called_once()
+
+    def test_auto_reconnect_only_still_calls_preconnect_cleanup(self, monkeypatch):
+        from mmrelay.meshtastic.connection import _connect_meshtastic_impl
+
+        class FakeAutoReconnectBLE:
+            def __init__(
+                self,
+                *,
+                address: str,
```

Ruff (0.15.11)

[warning] Unused method argument: `address`

(ARG002)